mojaloop/requirements.yaml

```diff
@@ -32,3 +32,13 @@ dependencies:
   version: 8.7.1
   repository: "file://../transaction-requests-service"
   condition: transaction-requests-service.enabled
+- name: finance-portal
+  version: 8.6.0
+  repository: "file://../finance-portal"
+  condtion: finance-portal.enabled
+- name: nginx-ingress
+  repository: https://kubernetes-charts.storage.googleapis.com
+  version: ~1.24.7
+- name: settlement
```

Comment:
Please confirm why this was added?

It should already be included as part of central chart as the `central-settlement` helm chart.